sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/specialized/DataLakeLeaseAsyncClient.java

```diff
@@ -255,7 +255,7 @@ public Mono<Integer> breakLease() {
      * <pre>
      * Integer retainLeaseInSeconds = 5;
      * RequestConditions modifiedRequestConditions = new RequestConditions&#40;&#41;
-     *     .setIfUnmodifiedSince&#40;OffsetDateTime.now&#40;&#41;.minusDays&#40;3&#41;&#41;;
+     *     .setIfModifiedSince&#40;OffsetDateTime.now&#40;&#41;.minusDays&#40;3&#41;&#41;;
```

Comment:
This changes the semantics of when the lease will be released. We should document what this sample is demonstrating. Is this trying to release the lease if it was recently modified (current version) or not release the lease if it was recently modified (previous version)?

Reply:
I'll add additional information to the sample. This changes it so it works when attempting to run the sample by changing the precondition to check "Acquire/renew the lease if the file has been modified in the last 3 days".